korman/exporter/material.py

```diff
@@ -166,7 +166,8 @@ def _export_texture_slot(self, bo, bm, hsgmat, slots, idx):
         if slot.use_stencil:
             hsgmat.compFlags |= hsGMaterial.kCompNeedsBlendChannel
             state.blendFlags |= hsGMatState.kBlendAlpha | hsGMatState.kBlendAlphaMult | hsGMatState.kBlendNoTexColor
-            state.clampFlags |= hsGMatState.kClampTexture
+            if slot.texture and slot.texture.type == "BLEND":
+                state.clampFlags |= hsGMatState.kClampTexture
```

Comment:
Do you mean whether the user can choose to clip or not his stencil texture ? Since the stencil is exported as an image afterwards, the "Extension" option in image mapping works just as intended.
My bad about `slot.texture`, I'll correct it.